### PR TITLE
Issue #32 - Add link to Directory profile to correct answers

### LINKED ISSUE #32: When you guess correctly, make the person's name a link to their directory profile
Quick way to learn a bit more about somebody else. Having that quick context and linking somebody's name and face with 'likes Ruby' from their profile/bio facilitates learning.

This should be a quick one as we already have all the information to do this. Somebody's profile has the format https://www.recurse.com/directory/[person ID]-[first name]-[last name].

This could be combined with #29 as the same React component needs to be modified on both. #29 requires some additional changes in React and on the SQL query side but changes are not that involved.

### PR DESCRIPTION
**Pull request by jaryncolbert**
This commit resolves #32 by composing a link as formatted in the original issue -  `https://www.recurse.com/directory/[person ID]-[first name]-[last name]`. However, I wasn't sure how to test this with real data to look at edge cases for RCers with multiple names. 

Also, per the [API docs](https://github.com/recursecenter/wiki/wiki/Recurse-Center-API), I wonder if `profile.slug` is a better field to use or if it already accounts for unique first/last name cases. However, this field isn't currently in the database, so it would require a backend change.  

Looking forward to talking through this PR further!

### CODE REVIEW

File: src/components/guess.js [hunk 1]
```diff
@@ -6,9 +6,14 @@ class Guess extends Component {
     const guess = this.props.guess;
     if (guess.get('status') === "correct") {
       let person = this.props.person;
+      let firstName = person.getIn(['0','first_name']);
+      let lastName = person.getIn(['0','last_name']);
+      let personId = person.getIn(['0','id']);
```
Comment (jasonaowen) [inline src/components/guess.js]: Hm, I'm getting `undefined` when I run this locally with Flask. I think this needs to be `person_id` instead:

```suggestion
      let personId = person.getIn(['0','person_id']);
```

**Comment by jaryncolbert** [comment]
I have a `git rebase` followed by `git push -f` gone wrong that I'm not sure how to recover from, lol. I'm going to close this PR and re-open another one so that the commit history isn't corrupted.

**Comment by jasonaowen** [comment]
This looks great to me! Thank you, @jaryncolbert :purple_heart:

File: src/components/guess.js (hunk 1 above)
Comment (jaryncolbert) [reply src/components/guess.js]: You're right; I was looking at the RC API fields, and not the local database table column names. I updated the references from `id` to `person_id` to fix this. 

Based on our conversation today, I also made the update to only use `person_id` inside the directory link. Maybe a future task could use the `slug` field given that we found a couple of edge cases like [JP](https://www.recurse.com/directory/3181-john-paul-jp-ashenfelter) where his directory link is different from what we would expect from his displayed name.
